velox/exec/Task.h

```diff
@@ -58,6 +68,7 @@ class Task : public std::enable_shared_from_this<Task> {
       core::PlanFragment planFragment,
       int destination,
       std::shared_ptr<core::QueryCtx> queryCtx,
+      ExecutionMode mode,
```

Comment:
You might need to have velox backward compatibility macro for create method used by Prestissimo